cli/unregister/unregister.go

```diff
@@ -288,3 +302,65 @@ func RemoveServiceContainers() error {
 		return fmt.Errorf(err_string)
 	}
 }
+
+// backupEventLogs loads eventlogs from anax.db, marshals them into the JSON format
+// and saves the bkp file into horizon folder with name of backup time
+func backupEventLogs() error {
```

Comment:
Please call anax REST API /eventlog to get the event logs and then save it to a text file.  Check this function to see how to get event logs from anax. https://github.com/open-horizon/anax/blob/master/cli/eventlog/eventlog.go#L66